docs/codebase.md

```diff
@@ -48,14 +48,17 @@ File Descriptions
 * Scrollbar.js
   * Renders the horizontal and vertical scrollbars for the table
   * Used by FixedDataTable.js
-* ColumnResizerLine.js
-  * Renders the line displayed while resizing a column
-  * Used by FixedDataTable.js
-* FixedDataTableColumnReorderHandle.js
-  * Renders the knob used to reorder a column
+* ResizeReorderCell.js
+  * Renders ReorderHandle and ResizerKnob
```

Comment:
Mention that this renders a cell that can be used resizing/reordering the columns

Reply:
Done